.ci/pytorch/build.sh

```diff
@@ -197,6 +197,12 @@ if [[ "$BUILD_ENVIRONMENT" == *-bazel-* ]]; then
   BAZEL_MEM_LIMIT="--local_ram_resources=HOST_RAM*.8"
   BAZEL_CPU_LIMIT="--local_cpu_resources=HOST_CPUS-1"
 
+  # libfmt provides conflicting bazel files against ours,
```

Comment:
@vors does that sound ok to you?

Reply:
can you try this pattern?

https://github.com/pytorch/pytorch/blob/e52786f3d177a7ca5d490a516cf52e236ef072cb/WORKSPACE#L246-L254

Reply:
cc @cyyever can you try that? :)